### DISCUSSION TITLE
I caught a block, but the payment went to another wallet

### DISCUSSION THREAD

**Discussion by RUPOOLprojectOff** [Chia-Network/chia-blockchain#8834] [Support]
Good afternoon, I am the admin of RUPOOLproject, already from the second user 0.25 xch went to another wallet and they cannot collect their reward, help to solve the problem!

Reply in the comments, I will send any information you need, including wallets

Thanks in advance for your help, we are with you as much as possible and are looking forward to Defi and smartcontracts

**Comment by plot14** [comment]
I have the same problem, I think we have all been defrauded. and there is no pre and post execution assistance for the chia project.

possible we can not go back to the various addresses and understand where the 0.25 xch end up?
